ecosystem/sep-0024.md

```diff
@@ -190,6 +192,73 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allows payments to accounts that do not have a trust-line for the asset sent.
+
+**Wallets**: It's mandatory for wallets to support Claimable Balances and should poll for their users' received Claimable Balances so that users can "claim" them.
+
+**Anchors**: It's optional for anchors to use submit Claimable Balances as deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
+
+**Wallet Claimable Balance Flow**
+
+Wallets will still follow the [typical deposit flow](#Basic-Wallet-Implementation).
+The POST request however will extend to include a `claimable_balance_supported` parameter
+
+**Wallet Claimable Balance Polling**
```

Comment:
Leigh also commented on this section, but the way I see it there are two methods for Wallets to detect claimable balances:

1. Checking Horizon
2. Checking the anchor's `GET /transaction(s)` endpoint

With the Horizon option, they could poll on demand or stream transactions to the users Stellar account.

Reply:
Yeah thats a cleaner way to split the options.